hwdb/parse_hwdb.py

```diff
@@ -112,6 +112,8 @@ def property_grammar():
              ('ID_INPUT_TOUCHPAD_INTEGRATION', Or(('internal', 'external'))),
              ('XKB_FIXED_LAYOUT', STRING),
              ('XKB_FIXED_VARIANT', STRING),
+             ('KEYBOARD_LED_NUMLOCK', Literal('0')),
+             ('KEYBOARD_LED_CAPSLOCK', Literal('0')),
```

Comment:
LGTM.

I wonder if we should allow `KEYBOARD_LED_NUMLOCK=1` and `KEYBOARD_LED_CAPSLOCK=1` for completeness.